ee/clickhouse/queries/util.py

```diff
@@ -133,5 +134,6 @@ def deep_dump_object(params: Dict[str, Any]) -> Dict[str, Any]:
     return params
 
 
+@lru_cache(maxsize=100000)
```

Comment:
100000? Why that?

If you want unlimited, then maxsize=None is more appropriate.

Reply:
that makes sense actually, I should just use the `@cache`

Reply:
Actually, I put a maxsize earlier because I was worried about it OOM'ing, and without a maxsize, there's no protection?

Reply:
@macobo are you aware of anything builtin to protect the cache from getting huge without a maxsize? Didn't find anything like this, so curious why would we use this without a maxsize?

Reply:
Will merge for now, and will ping you about this tomorrow so it doesn't get lost in the weeds :)